verification/cbmc/include/make_common_data_structures.h

```diff
@@ -38,6 +39,14 @@ bool aws_cryptosdk_edk_list_elements_are_bounded(const struct aws_array_list *co
 void ensure_cryptosdk_edk_list_has_allocated_list(struct aws_array_list *list);
 void ensure_cryptosdk_edk_list_has_allocated_list_elements(struct aws_array_list *list);
 
+/* Allocates the members of the header and ensures that internal pointers are pointing to the correct objects. */
+struct aws_cryptosdk_hdr *ensure_nondet_hdr_has_allocated_members();
+/* Determines if the members of the header are bounded. */
+bool aws_cryptosdk_hdr_members_are_bounded(
+    const struct aws_cryptosdk_hdr *const hdr, const size_t max_edk_item_size, const size_t max_item_size);
```

Comment:
You don't need to use `const` in pointers, so `const struct aws_cryptosdk_hdr *hdr` is enough.

Reply:
Fixed!